code/game/machinery/doppler_array.dm

```diff
@@ -183,6 +183,11 @@
 	records += R
 	//Update to viewers
 	ui_update()
+
+	for(var/mob/living/carbon/human/H in oviewers(src))
+		if(H.client)
+			INVOKE_ASYNC(H.client, /client.proc/increase_score, /datum/award/score/bomb_score, H, orig_light_range)
```

Comment:
Should this be for everyone watching, or just the person doing the signaler/detonation method

Reply:
Bombs of sufficiently high radius are often a team effort, and I don't want to shaft anyone that helps out in favor of whoever pushes the button.